Updated content based on feedback from Python team (#4562)
microsoft/vscode-docs: docs/python/pytorch-support.md

To start a TensorBoard session, open the **Command Palette** (`kb(workbench.action.showCommands)`) and search for the command `Python: Launch TensorBoard`. Afterwards, you will be prompted to select the folder where your TensorBoard log files are located. By default, VS Code uses your current working directory and automatically detects your TensorBoard log files within any subdirectories, but you can also specify your own directory. VS Code will then open a new tab with TensorBoard and manage its lifecycle as you work.

You can also use the setting `python.tensorboard.logDirectory` to set a default TensorBoard log directory for your folder/workspace.

## PyTorch Profiler integration with VS Code

Along with TensorBoard, VS Code and the Python extension also integrate the PyTorch Profiler, allowing you to better analyze your PyTorch models in one place. For more information about the profiler, see the [PyTorch Profiler documentation](https://pytorch.org/blog/introducing-pytorch-profiler-the-new-and-improved-performance-tool/).

![PyTorch Profiler integration with VS Code](images/pytorch-support/pytorch-profiler.png)

## IntelliSense for PyTorch through the Pylance language server

> Through the Python editing experience in VS Code powered by Pylance provides completions and other rich features for PyTorch. For the best experience, update PyTorch to 1.8.1 to get improved completions for submodules, such as nn, cuda and optim.

Reviewer: ```suggestion
The Python editing experience in VS Code, enhanced with the power of Pylance, provides completions and other rich features for PyTorch. For the best experience, update PyTorch to 1.8.1 to get improved completions for submodules, such as nn, cuda and optim.
```
